### PR TITLE
chore: rename Full node to Bridge node

### LINKED ISSUE #250: node: Clean up node types
### Summary

After devnet, we need to do hygiene work in our node types

### Gain

This is beneficial for our upcoming tool that will be creating in-memory network and deploy nodes to test against

### Evidence

Code-base node part

### Implementation ideas

_No response_

### Urgency

Medium

### PR DESCRIPTION
**Pull request by Wondertan**
Simple renaming.

Closes #250. I decided not to introduce a new Full node type yet as there are still things like #179 and #181 that need to be implemented first and maybe even #223

### CODE REVIEW

File: CHANGELOG-PENDING.md [hunk 1]
```diff
@@ -6,12 +6,11 @@ Month, DD, YYYY
...
 ### BREAKING CHANGES
 
-- [go package] (Link to PR) Description @username
-[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
+- [chore: rename Full node to Bridge node #294](https://github.com/celestiaorg/celestia-node/pull/294) [@Wondertan](https://github.com/Wondertan)
 
 ### FEATURES
 
-- [go package] (Link to PR) Description @username
+- [feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
```
Comment (renaynay) [inline CHANGELOG-PENDING.md:13]: why is this included in this PR?

File: node/node.go [hunk 2]
```diff
@@ -55,10 +55,6 @@ type Node struct {
 	// p2p protocols
 	PubSub *pubsub.PubSub
 	// BlockService provides access to the node's Block Service
-	// TODO @renaynay: I don't like the concept of storing individual services on the node,
-	// TODO maybe create a struct in full.go that contains `FullServices` (all expected services to be running on a
-	// TODO full node) and in light, same thing `LightServices` (all expected services to be running in a light node.
-	// TODO `FullServices` can include `LightServices` + other services.
```
Comment (renaynay) [inline node/node.go:61]: why did you remove this TODO? services are still stored not uniformly on the node: 

```
	BlockServ  *block.Service  `optional:"true"`
	ShareServ  share.Service   // not optional
	HeaderServ *header.Service // not optional
```

File: node/type.go [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-// Type defines the Node type (e.g. `light`, `full`) for identity purposes.
+// Type defines the Node type (e.g. `light`, `bridge`) for identity purposes.
 // The zero value for Type is invalid.
 type Type uint8
 
 const (
-	// Full is a full-featured Celestia Node.
-	Full Type = iota + 1
+	// Bridge is a Celestia Node that bridges Celestia consensus network and DA network.
+	// It maintains trusted channel/connection to Celestia Core Node using core.Client.
```
Comment (renaynay) [inline node/type.go:9]:
```suggestion
	// Bridge is a Celestia node that bridges the Celestia consensus network and data availability network.
	// It maintains a trusted channel/connection to a Celestia Core node via the core.Client API.
```

File: CHANGELOG-PENDING.md (hunk 1 above)
Comment (Wondertan) [reply CHANGELOG-PENDING.md:13]: It was placed in the wrong place. Should be fixed in https://github.com/celestiaorg/celestia-node/pull/280 and I will remove the change here while rebasing.

File: node/node.go (hunk 2 above)
Comment (Wondertan) [reply node/node.go:61]: > services are still stored not uniformly on the node.

TODO text is not related to the quote below, but to the fact that they are _stored on__ the node.

> why did you remove this TODO?

For you, you see this removal and create an appropriate issue. The TODO is not really a TODO, but an opinion spread onto four lines. Create an issue with the concern if still appropriate.

File: CHANGELOG-PENDING.md (hunk 1 above)
Comment (renaynay) [reply CHANGELOG-PENDING.md:13]: 👍🏻

**Review by renaynay — DISMISSED** [review]

File: cmd/celestia/bridge.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+func init() {
+	bridgeCmd.AddCommand(
+		cmd.Init(repoFlagName, node.Bridge),
+		cmd.Start(repoFlagName, node.Bridge),
+	)
+	bridgeCmd.PersistentFlags().StringP(
+		repoFlagName,
+		repoFlagShort,
+		"~/.celestia-bridge",
```
Comment (adlerjohn) [inline cmd/celestia/bridge.go:18]: This is the default directory for the node's persistent files, right? If we're building something that isn't just a scappy devnet I think this should be a constant instead of a hard-coded string.

Comment (Wondertan) [reply cmd/celestia/bridge.go:18]: @adlerjohn, I agree. Is it ok if I do this in a separate PR where I will do some flag changes? This is a bit unrelated to this one.

Comment (adlerjohn) [reply cmd/celestia/bridge.go:18]: Yeah sure, just track it in an issue.

**Review by adlerjohn — APPROVED** [review]

**Review by renaynay — APPROVED** [review]
